modules/installation-about-disconnected.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_disconnected/installing-disconnected-preparations.adoc
+
+[id="installation-about-disconnected_{context}"]
+= About disconnected installations
+
+In {product-title} {version}, you can perform an installation that does not
+require an active connection to the internet to obtain software components. You
+complete a disconnected installation on only infrastructure that you provision,
+not infrastructure that the installer provisions, so your platform selection is
+limited.
+// maybe point out that you can follow the bare metal installation rules on supported hardware and link to the matrix
+
+If you choose to perform a disconnected installation on a cloud platform, you
+still require access to its cloud APIs and will still require internet access
+to use your cluster. Depending on your network, you might require less internet
+access for an installation on bare metal hardware or on VMware vSphere.
+
+To complete a disconnected installation, you must create a repository that
+mirrors the contents of the {product-title} registry and contains the
+installation media. You can create this mirror on a bastion host, which can
+access both the internet and your closed network, or by using other methods
+that meet your restrictions.
+
+[IMPORTANT]
+====
+Because disconnected installations are always on user-provisioned infrastructure,
+always complete a standard user-provisioned infrastructure installation before
+you attempt a disconnected installation.
```

Comment:
You could do a disconnected vSphere install, right?  And there's also at least some work on [installer-provisioned metal][1], although I'm not sure if that's supported in 4.2 or not.  So I think we want to soften this, but I'm not sure what we want the new wording to be ;).

[1]: https://github.com/openshift/installer/blob/master/docs/user/metal/install_ipi.md

Reply:
vSphere is technically user-provisioned infrastructure, right?